app/konnect/dev-portal/register-and-create-app.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Register and create an application as a developer
+---
+
+This guide explains how developers can get started with the developer platform by registering and creating an application. 
+
+The Dev Portal enables developers to integrate applications with APIs from third-party providers. For instance, if you are developing for a company that requires access to two distinct APIs from a third-party provider, you would need to set up an application within the Dev Portal that consumes both APIs.
+
+The following diagram shows how you can register your app in Dev Portal:
+
+{% mermaid %}
+flowchart TD
+    A[Sign up for Dev Portal] --> B(Create an application)
+    B --> C(Add products to the application)
+    C --> D(Create application credentials)
+    D --> |Generate in Dev Portal| E[Use application] 
```

Comment:
From the developers perspective do they really need to worry about D? B (creating an application) will be informed by their Dev Portal admin--they'll tell them hey use key auth or OIDC or DCR and follow these steps. After that, they don't need to worry about application auth/creation

Reply:
@c3pko Do developers still need to create credentials if the application uses OIDC or DCR? I think I still might be confused about credentials and auth for developers.

Reply:
I think this comment was in reference to needing to select or create an auth method? I don't see anything about auth methods so we can disregard my comment maybe I misunderstood the original text